scripts/generate-integration-pages.ts

```diff
@@ -62,38 +64,48 @@ class IntegrationPagesBuilder {
 		}
 	}
 
+	async #getSingleIntegrationData({ packageName, pkgJsonURL, readmeURL }: { packageName: string; pkgJsonURL: string; readmeURL: string }): Promise<IntegrationData> {
+		const { name, keywords } = await githubGet({
+			url: pkgJsonURL,
+			githubToken: this.#githubToken,
+		});
+		const category = keywords.includes('renderer')
+			? 'renderer'
+			: keywords.includes('astro-adapter')
+			? 'adapter'
+			: 'other';
+		const i18nReady = (!this.#i18nNotReadyIntegrations.has(packageName)).toString();
+		const readme = await (await fetch(readmeURL)).text();
+		return { name, category, readme, srcdir: packageName, i18nReady };
+	}
+
 	/**
 	 * Collect data for each official Astro integration.
 	 */
 	async #getIntegrationData(): Promise<IntegrationData[]> {
 		// Read all the packages in Astro’s integrations directory.
 		const url = `https://api.github.com/repos/${
 			this.#sourceRepo
-		}/contents/packages/integrations?ref=${this.#sourceBranch}`;
+		}/contents/${this.#sourcePath}?ref=${this.#sourceBranch}`;
 		const packages: { name: string }[] = await githubGet({ url, githubToken: this.#githubToken });
 
 		return await Promise.all(
 			packages
 				.filter((pkg) => !this.#deprecatedIntegrations.has(pkg.name))
-				.map(async (pkg) => {
+
+				.map(async(pkg) => {
 					const pkgJsonURL = `https://raw.githubusercontent.com/${this.#sourceRepo}/${
 						this.#sourceBranch
-					}/packages/integrations/${pkg.name}/package.json`;
+					}/${this.#sourcePath}/${pkg.name}/package.json`;
 					const readmeURL = `https://raw.githubusercontent.com/${this.#sourceRepo}/${
 						this.#sourceBranch
-					}/packages/integrations/${pkg.name}/README.md`;
-					const { name, keywords } = await githubGet({
-						url: pkgJsonURL,
-						githubToken: this.#githubToken,
+					}/${this.#sourcePath}/${pkg.name}/README.md`;
+
+					return this.#getSingleIntegrationData({
+						packageName: pkg.name,
+						pkgJsonURL,
+						readmeURL
 					});
-					const category = keywords.includes('renderer')
-						? 'renderer'
-						: keywords.includes('astro-adapter')
-						? 'adapter'
-						: 'other';
-					const i18nReady = (!this.#i18nNotReadyIntegrations.has(pkg.name)).toString();
-					const readme = await (await fetch(readmeURL)).text();
-					return { name, category, readme, srcdir: pkg.name, i18nReady };
 				})
 		);
 	}
```

Comment:
`packages/integrations` needs replacing with the new `sourcePath` option in `#processReadme()` below (sorry GitHub won’t let me comment on that line).

Reply:
Good catch, updated this.